src/main/java/com/group1/stagesWs/controller/OffreController.java

```diff
@@ -36,4 +36,11 @@ public OffreController(OffreService service) {
                 .map(ResponseEntity::ok)
                 .orElse(ResponseEntity.status(HttpStatus.CONFLICT).build());
     }
+
+    @PostMapping("/{id}/apply")
+    public ResponseEntity<Offre> applyForOffre(@PathVariable int id, @RequestBody String email) {
```

Comment:
Why is there a request body of string email in parameters?

Reply:
Figured it would be easier to send the email than to send the etudiant object